packages/wasi/src/wasi.ts

```diff
@@ -714,24 +714,32 @@ class WASI {
       fd_read: wrap(
         (fd: number, iovs: number, iovsLen: number, nread: number) => {
           const stats = CHECK_FD(fd, WASI_RIGHT_FD_READ);
+          const IS_STDIN = stats.real === 0;
           let read = 0;
           outer: for (const iov of getiovs(iovs, iovsLen)) {
             let r = 0;
             while (r < iov.byteLength) {
-              const rr = fs.readSync(
-                stats.real,
-                iov,
-                r,
-                BigInt(iov.byteLength - r) + stats.offset
+              let length = iov.byteLength - r;
```

Comment:
Is this branch not up to date with master? 🤔

Reply:
Oh! I just accidentally pushed a fix for #135 ...but it should be good to go as well!